Stellar-ledger-entries.x

```diff
@@ -520,20 +520,13 @@ case CONFIG_SETTING_TYPE_UINT32:
 
```

Comment:
I think `ConfigSettingType` and `ConfigSetting` can be removed as they are not being used any more.

Reply:
Not sure I understand; why are these not being used?

Reply:
These lines https://github.com/stellar/stellar-xdr/pull/56/files#diff-bd931d19b88f0dfe6138c05c73d5f2d2d4c6ba244d2764d11fb9443f21fda05aR510-R519
 are outdated.

Reply:
I'm pretty sure they're deleted, maybe you are looking at some outdated version of the file (FWIW I see your comment attached to the initial version and not the changed one)